packages/cli/src/index.ts

```diff
@@ -352,16 +376,17 @@ program
 
 program
   .command("transfer <marketId> <sharesIndex> <to> <amount>")
+  .option(
+    "--seed <string>",
+    "The signer's seed. Default is:",
+    "clean useful exotic shoe day rural hotel pitch manual happy inherit concert"
+  )
   .option(
     "--endpoint <string>",
     "The endpoint to connect the API to.",
     "wss://bp-rpc.zeitgeist.pm"
   )
-  .option(
-    "--seed <string>",
-    "The signer's seed. Default is `//Alice`.",
-    "clean useful exotic shoe day rural hotel pitch manual happy inherit concert"
-  )
+  // TODO: check if these params really should be string!
```

Comment:
Can you open an issue on GitHub for this so the TODO doesn't get lost as a comment?